docs/reference/analysis/analyzers/lang-analyzer.asciidoc

```diff
@@ -1588,6 +1589,55 @@ PUT /russian_example
 <2> This filter should be removed unless there are words which should
     be excluded from stemming.
 
+[[serbian-analyzer]]
+===== `serbian` analyzer
+
+The `serbian` analyzer could be reimplemented as a `custom` analyzer as follows:
+
+[source,console]
+----------------------------------------------------
+PUT /serbian_example
+{
+  "settings": {
+    "analysis": {
+      "filter": {
+        "serbian_stop": {
+          "type":       "stop",
+          "stopwords":  "_serbian_"
+        },
+        "serbian_keywords": {
+          "type":       "keyword_marker",
+          "keywords":   ["пример"]
+        },
+        "serbian_stemmer": {
+          "type":       "stemmer",
+          "language":   "serbian"
+        }
+      },
+      "analyzer": {
+        "rebuilt_serbian": {
+          "tokenizer":  "standard",
+          "filter": [
+            "lowercase",
+            "serbian_stop",
+            "serbian_keywords",
+            "serbian_stemmer",
+            "serbian_normalization"
+          ]
+        }
+      }
+    }
+  }
+}
+----------------------------------------------------
+// TEST[s/"serbian_keywords",//]
+// TEST[s/\n$/\nstartyaml\n  - compare_analyzers: {index: serbian_example, first: serbian, second: rebuilt_serbian}\nendyaml\n/]
+
+<1> The default stopwords can be overridden with the `stopwords`
+or `stopwords_path` parameters.
```

Comment:
I think we need anchors for these call-outs in L1606 and L1610 respectively

Reply:
Yes, you are right. I added the anchors in the required places in the new commit.